### PR TITLE
Added Thank you for Contributing in telugu.

### PR DESCRIPTION
**Pull request by AshwanthKumar123**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://github.com/AshwanthKumar123/gci18.fossasia.org)

- Fixes #569  

## Description / Changes
Added 'Thank you for Contributing' in telugu.

### Before:
before adding thank you for contributing in telugu
### After:
after adding thank you for contributing in telugu

- - - - - - - - - - - -

### CODE REVIEW

**Comment by AshwanthKumar123** [comment]
UPDATED

File: js/thankyou.js [hunk 1]
```diff
@@ -31,6 +31,9 @@
 'സംഭാവന ചെയ്തത്തിന് നന്ദി',
 'Takk fyrir þitt framlag!',
 'Tibi gratias ago pro tui contributions!'
+'సహకరించినందుకు ధన్యవాదాలు'
+
```
Comment (jogendra) [inline js/thankyou.js]: remove extra line

File: _data/mentors.yml [hunk 2]
```diff
@@ -9,6 +9,11 @@
 #   facebook: Mentor's Facebook ID
 #   lat: Mentor's Latitude
 #   lng: Mentor's Longitude
+
+- name: Ayush Gupta
```
[the remark this answers is not in the document]
Comment (AshwanthKumar123) [reply _data/mentors.yml]: it is already merged

**Comment by AshwanthKumar123** [comment]
i had added preview link already

**Comment by pythongiant** [comment]
@Ashwanth7230 thats not what we mean by a preview link. we need a gh-page link 
github.io
check this out
and... please refrence issue numbers properly

**Comment by AshwanthKumar123** [comment]
> @Ashwanth7230 thats not what we mean by a preview link. we need a gh-page link
> github.io
> check this out
> and... please refrence issue numbers properly

done!

**Comment by AshwanthKumar123** [comment]
Updated again

**Comment by pythongiant** [comment]
no

**Comment by pythongiant** [comment]
www.github.io

File: _data/mentors.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (abishekvashok) [reply _data/mentors.yml]: Please rebase your branch correctly

**Pull request closed without merging**